src/unit_tests/shared/test_agent_op.c

```diff
@@ -152,6 +153,65 @@ static void test_create_sendsync_payload(void **state) {
     cJSON_Delete(payload);
 }
 
+static void test_get_agent_group(void **state) {
+    char* group;
+    int agent_id = 1;
+    int wdb_sock = -1;
+    size_t size = OS_SIZE_65536 + 1;
+    int res;
+
+    os_calloc(size, sizeof(char), group);
+    snprintf(group, size, "%s","default");
```

Comment:
In this test **group** is being used as an output buffer (as it is normally used in the methods that call a **test_get_agent_group**) and also as an input buffer to create the json objects. I think we should only use it as an output buffer, and create the json objects with a raw string in the creation.